### PR TITLE
Check in L2 lib code

### PR DESCRIPTION
**Pull request by hongzhang-github**

### CODE REVIEW

**Comment by hongzhang-github** [comment]
This is the first check-in for g11n-js-sdk including bug fixes for VIP-2444,VIP-2449 and VIP-2450.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "prepublish": "rimraf dist && tsc -p ./ --outDir dist/ && npm run build",
+        "jest": "jest -c jest.config.json",
+        "test": "npm run jest"
+    },
+    "keywords": [
+        "singleton",
+        "i18n",
+        "l10n",
+        "node",
+        "VIP"
```
Comment (songjiang320) [inline package.json:22]: It's better remove 'VIP' here

Comment (hongzhang-github) [reply package.json:22]: Ok, I'll remove it next time is there are no other issues currently.

Comment (hongzhang-github) [reply package.json]: https://github.com/hongzhang-github/singleton/commit/d852d11209b1ce8dd0e2e7d2307d3b458a6217b7

**Pull request merged**